app/jest/helpers.test.ts

```diff
@@ -3,12 +3,10 @@ import { isValidKeycloakURIDev, isValidKeycloakURIProd } from 'utils/validate';
 
 describe('kecloak URIs', () => {
   it('should validate the URI correctly', () => {
-    expect(isValidKeycloakURIDev('*')).toBe(false);
+    expect(isValidKeycloakURIDev('*')).toBe(true);
```

Comment:
Did we want to let all redirects through?

Reply:
yeah in dev we will allow * again, it was causing some issues

Reply:
only on the dev check, the prod check still doenst allow for it